examples/examples.py

```diff
@@ -7,7 +7,8 @@
 log = logging.getLogger(__name__)
 
 csv_path = "./examples/outputs"
-os.makedirs(csv_path, exist_ok=True)
+if csv_path:
+    os.makedirs(csv_path, exist_ok=True)
```

Comment:
csv_path is always defined above..
you should just remove the makedirs line entirely, as this is taken care of in outputs.py in line 58

Reply:
cool